src/main/java/io/vertx/core/DeploymentOptions.java

```diff
@@ -111,23 +120,43 @@ public DeploymentOptions setConfig(JsonObject config) {
     return this;
   }
 
+  /**
+   * How the verticle(s) should be deployed?
+   *
+   * @return the verticle thread mode
+   */
+  public ThreadMode getThreadMode() {
```

Comment:
I'd use `threadingModel` over `threadMode` all over the place (methods, fields, etc)

Reply:
good idea